examples/typescript-cdk-v2/lib/typescript_v2-stack.ts

```diff
@@ -2,6 +2,7 @@ import * as cdk from "aws-cdk-lib";
 import { Construct } from "constructs";
 import * as lambda from "aws-cdk-lib/aws-lambda";
 import { Function } from "aws-cdk-lib/aws-lambda";
+import * as aws_apigateway from "aws-cdk-lib/aws-apigateway";
```

Comment:
Can we import like this
```suggestion
import { RestApi, LambdaIntegration } from "aws-cdk-lib/aws-apigateway";
```
or name the * import `awsApiGateway` to avoid the snake_case?